central/networkgraph/service/service_impl.go

```diff
@@ -309,9 +312,68 @@ func (s *serviceImpl) getNetworkGraph(ctx context.Context, request *v1.NetworkGr
 			node.QueryMatch = true
 		}
 	}
+
+	if request.IncludePolicies {
+		err := s.enhanceWithNetworkPolicyIsolationInfo(ctx, graph)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to enhance graph nodes with Network Policy information")
```

Comment:
Shall we maybe return the graph anyway `return graph, errors.Wrap(err,..)`? That would be a bit more backwards compatible (if that makes sense at all). I have personally no opinion about this, but maybe you have?

Reply:
Yea, I guess that makes sense. I think it would be better to log a warning and then return the graph without an error object. I think returning an error here would result in a non-200 HTTP status code.

Reply:
Done

Reply:
🤔 I think this didn't get pushed.

Reply:
I think I lost the change while restoring/stashing changes. Re-implemented and pushed a new commit. Good catch @lvalerom